Cargo.toml

```diff
@@ -3,6 +3,15 @@ name = "ssi"
 version = "0.1.0"
 authors = ["Spruce Systems, Inc."]
 edition = "2018"
+license = "Apache-2.0 AND W3C-20150513 AND CC-BY-SA-3.0"
```

Comment:
Can you please explain the addition of "AND W3C-20150513 AND CC-BY-SA-3.0"? Thanks!

Reply:
That sounds like a great approach, maybe `ssi-contexts` or `ssi-schemas` as an omnibus channel for them, even ours? Better names welcome.

Reply:
Opened https://github.com/spruceid/ssi/issues/156.

Another possible name: `ssi-vocabs`. But I think I would go with `ssi-contexts`, at least until we need to bundle things other than JSON-LD contexts.

Reply:
`ssi-contexts` published: https://github.com/spruceid/ssi/pull/157#issuecomment-811953571. License for the `ssi` crate now remains `Apache-2.0`.